src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -698,7 +699,8 @@ prepareTxInfoRaw moneySource dstDistr = do
         coins = foldr1 unsafeAddCoin $ snd <$> dstDistr
     distr@(remaining, spendings) <- selectSrcAccounts coins notDstAccounts
     logDebug $ buildDistribution distr
-    bootEra <- gsIsBootstrapEra
+    epoch <- siEpoch <$> getCurrentSlotBlocking
```

Comment:
http://i.imgur.com/WWnfcJO.png

So let's use blocking version unless someone else enters this discussion and tells us not to do it.